network/Message.cpp

```diff
@@ -942,15 +916,29 @@ void ExtractJoinAckMessageData(const Message& msg, int& player_id,
     }
 }
 
-void ExtractTurnOrdersMessageData(const Message& msg, OrderSet& orders) {
+void ExtractTurnOrdersMessageData(const Message& msg, OrderSet& orders, bool& ui_data_available,
+                                      SaveGameUIData& ui_data, bool& save_state_string_available,
```

Comment:
dedent to align with `const`

Reply:
should probably break the parameters into one-per-line, or move the first down to the next line, as it goes too far to the right to read easily now

Reply:
Made it one-per-line.